wordpress-mobile/WordPress-iOS
# Stats > Latest Post Summary: don't show error message if there is no post

Fixes #15401 
WPKit PR: https://github.com/wordpress-mobile/WordPressKit-iOS/pull/371

This fixes an issue where an error was displayed for Latest Post Summary if the site had no posts. Now the Create Post prompt is displayed.

To test:
- On a site that has no posts, go to Stats > Insights.
- If necessary, add `Latest Post Summary` to Insights.
- Verify the Create Post message is displayed.

| Before | After |
|--------|-------|
| <img width="465" alt="Screen Shot 2021-03-11 at 4 03 15 PM" src="https://user-images.githubusercontent.com/1816888/110867776-94486b00-8284-11eb-8903-e55afa30330d.png"> | <img width="473" alt="Screen Shot 2021-03-11 at 4 06 06 PM" src="https://user-images.githubusercontent.com/1816888/110867784-99a5b580-8284-11eb-970a-cb7f6fa1e351.png"> |

PR submission checklist:
